core/src/test/scala/integration/kafka/api/SslAdminIntegrationTest.scala

```diff
@@ -125,6 +130,24 @@ class SslAdminIntegrationTest extends SaslSslAdminIntegrationTest {
     startSasl(jaasSections(List.empty, None, ZkSasl))
```

Comment:
Should we use `KafkaSasl` instead if zk is removed?

Reply:
Yes, but we can do this only after removing all zk tests in `BaseAdminIntegrationTest` and `SaslSslAdminIntegrationTest` as this test class extends SaslSslAdminIntegrationTest.

Should I remove zk in both 2 test class as I mentioned above ?

Reply:
> Should I remove zk in both 2 test class as I mentioned above ?

Maybe it can be a follow-up

Reply:
Just tried, and it turns out there isn't too much effort to remove zk test in those test classes, so I'll do this in the next commit, thanks.